src/main/java/com/internet/shop/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.internet.shop.service;
```

Comment:
```suggestion
package com.internet.shop.service.impl;
```
same as in the package Dao